src/components/Process/Header.tsx

```diff
@@ -229,6 +233,60 @@ const ProcessHeader = () => {
   )
 }
 
+// todo(kon): disable this cast when ready on the sdk
+type Census3StrategyTokenIcon = Census3StrategyToken & { iconURI: string }
+
+const GitcoinStrategyInfo = () => {
+  const { t } = useTranslation()
+  const { election } = useElection()
+
+  if (!election || (election && !election?.meta?.strategy)) return ''
```

Comment:
Do not return empty string, simply `return` if you don't want the component to be rendered.

```suggestion
  if (!election || (election && !election?.meta?.strategy)) return
```